README.md

```diff
@@ -2,12 +2,106 @@
 
 everything you need to know about dahlia... WISDOM.
 
-### bootable usb 
+## how to make a bootable usb(GRUB)(ALL DEVICES)
 
-see **[here](https://github.com/dahlia-os/documentation/blob/master/how%20to%20make%20a%20bootable%20usb.md/)** to make a bootable usb.
+### x86_64
 
-### dahlia in qemu 
+**64/32 uefi systems and 64 bios systems**
 
-see **[here](https://github.com/HexaOneOfficial/documentation/blob/master/run%20dahlia%20in%20qemu/EN.md)** EN version.
+1 download the latest **iso [here](https://github.com/HexaOneOfficial/dahliaos/releases/download/200630_2/DahliaOS200630_2.iso)**. 
 
-see **[here](https://github.com/HexaOneOfficial/documentation/blob/master/run%20dahlia%20in%20qemu/NL.md)** NL version.
+2 then flash the iso to your usb with **rufus [here](https://rufus.ie/)**.
+
+3 disable secure boot if you have that option.
+ 
+
+**32 bios systems**
+
+1 download the latest **iso [here](https://github.com/HexaOneOfficial/dahliaos/releases/download/200630_2/DahliaOS200630_2.32bit.iso)**.
+
+2 then flash the iso to your usb with **rufus [here](https://rufus.ie/)**.
+
+3 disable secure boot if you have that option.
+
+### raspberry pi 3/4
+
+coming soon
+
+## how to make a bootable usb(NON GRUB)(FEW DEVICES)
+
+Firstly, you wanna check if you have one of the few supported devices [here](https://github.com/HexaOneOfficial/documentation/blob/master/supported%20hardware%20non%20grub.md).
```

Comment:
please change the link adress to point to dahliaOS isntead of your fork

Reply:
everything is fixed